tutorials/identifying-first-day-no-snow/index.md

```diff
@@ -0,0 +1,516 @@
+slope for each pixel is calculated with year as the x variable and y as first
+DOY with no snow.
+
+```js
+// Calculate slope image.
+var slope = annualCol.sort('year').select(['year', 'calDoy'])
+  .reduce(ee.Reducer.linearFit()).select('scale');
+
+// Define visualization arguments.
+var visArgs = {
+  min: -1,
+  max: 1,
+  palette: ['b2182b', 'ef8a62', 'fddbc7', 'f7f7f7', 'd1e5f0', '67a9cf', '2166ac']};
+
+// Map it.
+Map.setCenter(-95.78, 59.451, 5);
+Map.addLayer(slope, visArgs, '2000-2019 first day no snow slope');
+```
+
+The result is a map where red is a negative slope (progressively earlier first
+DOY with no snow), white is 0, and blue is positive (progressively later
+first DOY with no snow). In southern Norway and Sweden, the trend in
+first DOY with no snow between 2002 and 2019 appears to be influenced by
+various factors. Coastal areas exhibit progressively earlier first DOY of no
+snow than inland areas; however, high variability in slopes can be observed
+around fjords and in mountainous regions. This map reveals the complexity of
+seasonal snow dynamics in these areas.
+
+**Note** that the goodness of fit is not measured here, nor is significance
+considered. Slope may indicate regional trends, but local trends should be
+investigated using a time series chart (see the next section). Inter-annual
+variability can be influenced by masked pixels, as described above.
+
+![per-pixel-time-series-slope-map.jpg](per-pixel-time-series-slope-map.jpg)<br>
+_Figure 4. Map representing the slope of first DOY with no snow cover between
+2000 and 2019. The slope represents the overall trend. Color grades from red
+to blue as the slope in DOY increases. White indicates areas of little to no
+change._
+
+###  Time series chart
+
+To visually understand the temporal patterns of the first date of no snow
+through time, we can display our results in a time series chart. In this case,
+we’ve defined a 500m radius circle around a point of interest and
+calculated the annual mean first DOY with no snow for pixels within the region.
+
+```js
+// Define an AOI.
+var aoi = ee.Geometry.Point(-94.242, 65.79).buffer(1e4);
+Map.addLayer(aoi, null, 'Area of interest');
+
+// Calculate annual mean DOY of AOI.
+var annualAoiMean = annualCol.select('calDoy').map(function(img) {
+  var summary = img.reduceRegion({
+    reducer: ee.Reducer.mean(),
+    geometry: aoi,
+    scale: 1e3,
+    bestEffort: true,
+    maxPixels: 1e14,
+    tileScale: 4,
+  });
+  return ee.Feature(null, summary).set('year', img.get('year'));
+});
+
+// Print chart to console.
+var chart = ui.Chart.feature.byFeature(annualAoiMean, 'year', 'calDoy')
+  .setOptions({
+    title: 'Regional mean first day of year with no snow cover',
+    legend: {position: 'none'},
+    hAxis: {
+      title: 'Year',
+      format: '####'
+    },
+    vAxis: {title: 'Day-of-year'}});
+print(chart);
+```
+
+As is evident by the displayed results, the first date of no snow was mostly
```

Comment:
Nit: "date"->"day" for consistency (date: 2020-02-01 vs day: 25)

Reply:
Done